issue2pr.py

```diff
@@ -9,10 +9,27 @@
 
 from __future__ import print_function
 
+import argparse
+import getpass
+import json
```

Comment:
Eh, I didn't put any version requirement on requests, so I'll leave this as is for now.